deploy/complete/helm-chart/provision/templates/service-instance-global.yaml

```diff
@@ -51,7 +51,7 @@ spec:
   clusterServiceClassExternalName: object-store-service
   clusterServicePlanExternalName: standard
   parameters:
-    name: {{ (printf "MuShop %s Catalogue Bucket" (randAlphaNum 4)) | quote }}
+    name: {{ (printf "MuShop_%s_Catalogue_Bucket" (randAlphaNum 4)) | quote }}
```

Comment:
nit: 
Because it's a bucket that's created, there's no need to add "Bucket" to the name. Also, preferred qualifier would be "media" instead of "catalogue"

Reply:
Just following the original on the DB: `name: {{ (printf "MuShop %s Db" (randAlphaNum 4)) | quote }}`. There's Db everywhere. 
<img width="663" alt="image" src="https://user-images.githubusercontent.com/551007/73596568-afc13e00-44e8-11ea-9a1a-97a53cb0935e.png">


Those are not generic media, just serve the catalogue images related to products. There's no web assets.
Propose to change to `MuShop_%s_Catalogue_Media_Images`

Reply:
or `MuShop_%s_Product_Images`